documentation/production-checklist.md

```diff
@@ -0,0 +1,62 @@
+# Production checklist
+
+For the service to be ready for end users, it must be reliable, performant and sustainable.
+
+## StatusCake
+This is the most essential monitoring as if it alerts, it means users cannot access the site. It monitors both [uptime](https://www.statuscake.com/features/uptime/) and [SSL certificate](https://www.statuscake.com/features/ssl/). Use the [terraform module](https://github.com/DFE-Digital/terraform-modules/blob/main/monitoring/statuscake/README.md) to configure it.
+
+## Multiple replicas
+By default the template deploys only 1 replica for each [kubernetes deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). This is not sufficient for production as if the container is unavailable, there is no other replica to serve the requests. It may be unavailable because of high usage or simply because the cluster is moving the container to another node. This will happen when the cluster version is updated.
```

Comment:
Do we add a comment on replica memory sizing? Default for prod is 1G but app memory usage should be checked before and after launch.